src/Mod/Fem/femtest/data/ratel/material_mooney_rivlin.yml

```diff
@@ -0,0 +1,10 @@
+dm_plex_filename: 
+
+bc:
+
+model: elasticity-mooney-rivlin-initial
+nu_smoother: 0.39
+mu_1: 0.5
+mu_2: 0.5
+E: 3200000.0
```

Comment:
These are redundant for Mooney-Rivlin, for which `mu_1` and `mu_2` are needed, and specify the linear response (`E`).